casper/src/main/scala/coop/rchain/casper/ReportingCasper.scala

```diff
@@ -105,18 +102,25 @@ object ReportingCasper {
   type RhoReportingRspace[F[_]] =
     ReportingRspace[F, Par, BindPattern, ListParWithRandom, TaggedContinuation]
 
-  def rhoReporter[F[_]: ContextShift: Monad: Concurrent: Log: Metrics: Span: Parallel: BlockStore: BlockDagStorage](
+  def rhoReporter[F[_]: Sync: ContextShift: Monad: Concurrent: Log: Metrics: Span: Parallel: BlockStore: BlockDagStorage](
```

Comment:
`Sync` and `Monad` are not needed here, they are part of `Concurrent` trait. `Sync[F].delay` can be instead `Concurrent[F].delay`.